app/models/spotlight/blacklight_configuration.rb

```diff
@@ -123,7 +123,7 @@ def blacklight_config
         config.view.embed!
         # This is blacklight-gallery's openseadragon partial
         unless config.view.embed.document_component
-          config.view.embed.partials ||= ['openseadragon']
+          config.view.embed.partials ||= ['openseadragon'] unless config.view.embed.embed_component
```

Comment:
Do we want to put something in the template that shows the usage of `embed_component`?

Reply:
No. Some future version of the `blacklight_gallery:install` generator will do that.